Dockerfile-slim.template

```diff
@@ -1,14 +1,33 @@
-FROM python:%%PYTHON_VERSION%%-slim
+FROM python:%%PYTHON_VERSION%%-slim-stretch
```

Comment:
Making this more explicit is a good idea.

Reply:
👍